config-model/src/test/java/com/yahoo/vespa/model/container/xml/AccessControlTest.java

```diff
@@ -44,6 +44,10 @@
             StateHandler.STATE_API_ROOT,
             ContainerCluster.ROOT_HANDLER_PATH);
 
+    public AccessControlTest() {
```

Comment:
Please add a comment explaining why Before is not used, e.g. "`isHosted`must be initialized before the Before method of the super class".

Reply:
Fixed